controller/lighting_unit.py

```diff
@@ -0,0 +1,140 @@
+from neopixel import NeoPixel
+from machine import Pin
+
+
+class LightingUnit:
+    """A class representing an LED lighting unit. It implements some basic
+    functionality necessary for controlling a hardware LED lighting unit,
+    such as those made up of addressable pixels (like the WS2812B) connected
+    in serial
+
+    :param outputPin: the pin ID for the controller's output to the lighting
+        unit
+    :type outputPin: int
+    :param pixelCount: the number of pixels in the lighting unit
+    :type pixelCount: int
+    """
+    def __init__(self, outputPin, pixelCount):
+        """Constructor method
+        """
+        self._pinOut = Pin(outputPin, Pin.OUT)
```

Comment:
Initialize this to 0.  You can do that in the parameter list:

`def __init__(self, outputPin: int = 0, pixelCount: int = 0`